samples/military_feature_schemas/Fields_Air.csv

```diff
@@ -0,0 +1,30 @@
+field_name,field_type,field_length,field_alias,nullability,field_domain,field_default,field_notes
+identity,Integer,4,Standard Identity,NULLABLE,Identity,,
+context,SmallInteger,2,Context,NULLABLE,Context,,
+symbolset,SmallInteger,2,Symbol Set,NULLABLE,Air,1,
+entity,Integer,4,Air Vehicle Type Icon,NULLABLE,Air_Entities,,
+modifier1,Integer,4,Modifier 1,NULLABLE,Air_Modifier_Ones,,
+modifier2,Integer,4,Modifier 2,NULLABLE,Air_Modifier_Twos,,
+indicator,Integer,4,"HQ, Task Force, Feint/Dummy",NULLABLE,HQTFFD,,
+operationalcondition,Integer,4,Operational Condition,NULLABLE,Operational_Condition_Amplifier,,
+uniquedesignation,String,30,Unique Designation,NULLABLE,,,
+additionalinformation,String,50,Additional Information,NULLABLE,,,
+staffcomment,String,50,Staff Comments,NULLABLE,,,
+credibility,SmallInteger,2,Credibility,NULLABLE,Credibility,,
+reliability,String,2,Reliability,NULLABLE,Reliability,,
+idmode,String,15,IFF/SIF/AIS,NULLABLE,,,
+type,String,24,Type of Equipment,NULLABLE,,,
+direction,SmallInteger,2,Direction,NULLABLE,Direction,0,
+speed,SmallInteger,2,Speed,NULLABLE,,,
+speedunit,String,3,Speed Unit,NULLABLE,SpeedUnits,,
```

Comment:
@abouffard - in general go ahead and add an issue in the military-features-data repo before adding new fields to the GDB specification just to make sure everyone concurs (there is already a PR recommendation to document these changes there) - specifically for this one, since it seems to be a bit of a problem with distribution include as much information as allowed though if the format is LIMDIS it is obviously not something we can use/include (for instance we would need some examples & how the 2 fields should be combined to make the label also)